recipes/example/meta.yaml

```diff
@@ -1,60 +0,0 @@
-# Note: there are many handy hints in commenets in this example -- remove them when you've finalized your recipe
```

Comment:
Ah yeah good catch, thanks.  I usually will run the docker build script (`run_docker_build.sh` ) locally to make sure that this will at least build on circle-ci before I waste CI cycles on conda-forge.  But running that script locally will delete the example recipe and I have about a 50% success rate of committing that deleted example up in a PR. :disappointed: